src/test_pages/BasePage.ts

```diff
@@ -41,6 +41,14 @@ export abstract class BasePage {
     }
   }
 
+  protected async verifySelectionEnable(locator: Locator, isEnable: boolean, timeout?: number) {
```

Comment:
setting defaults here can be nice
```suggestion
  protected async verifySelectionEnable(locator: Locator, isEnable=true, timeout=5000) {
```